forem/forem
# Add support for WebP on upload

## What type of PR is this? (check all applicable)


## Description

Add support for WebP format on upload, to all uploaders.

Also removes `dng` file extension which we've never actually supported anyway (as it's not supported being a RAW format).

Closes https://github.com/thepracticaldev/dev.to/issues/9118

**TODO:**

- [x] add support for webp to the code
- [ ] add support for webp on Travis
- [ ] add support for webp in Docker image
- [ ] add support for webp on Heroku

## Related Tickets & Documents

https://github.com/thepracticaldev/dev.to/issues/9118

## QA Instructions, Screenshots, Recordings

- Install webp support on your computer:
  - macOS: `brew info imagemagick` make sure it has `webp` installed in the list of `Required`
- Download a WebP image from https://developers.google.com/speed/webp/gallery1 and upload it

## Added tests?
